### ISSUE TITLE
Breadcrumb: Selected Aria State is Missing

### ISSUE THREAD

**Issue by scurker** [dequelabs/cauldron#1215]
Auditor Issue: https://axeauditor.dequecloud.com/test-run/87ff92fc-25f1-11ee-a855-1b2298d1b894/issue/7cf7860a-5cfe-11ee-933f-07f04e8012cd?activeTab=dt-issue&page=0&sortField=ordinal&sortDir=asc&row=3&issuesCount=&issuesCount=&filter=

## Description

State: Selected state of the element is missing or incorrect

Element description:  "Four" - aria current is missing for the current location 
Element location: In the "Basic" and "Custom Separators" examples

## Dev Notes

We should add `aria-current="true"` and toss up a note that as a best practice, the author should provide an [`aria-current`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current) value where necessary.
